tx/textual/valuerenderer/message_test.go

```diff
@@ -51,3 +56,31 @@ func TestMessageJsonTestcases(t *testing.T) {
 		})
 	}
 }
+
+func TestRepeatedJsonTestcases(t *testing.T) {
```

Comment:
I propose to move this test to `repeated_test.go` (like initially in your PR).

I feel like message and repeated value renderers are conceptually 2 different renderers (2 separate sections in the spec), but it's an implementation detail that we put them together in the message value renderer. But the test files can be separate

Reply:
Sounds good

Reply:
Done in 22f9b6c